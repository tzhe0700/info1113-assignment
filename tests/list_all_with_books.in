ADD COLLECTION spellbooks.csv
LIST ALL
EXIT

