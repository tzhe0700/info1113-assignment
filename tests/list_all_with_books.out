17 spellbooks successfully added.
The Standard Book of Spells Grade 1 (Miranda Goshawk)
The Standard Book of Spells Grade 2 (Miranda Goshawk)
The Standard Book of Spells Grade 3 (Miranda Goshawk)
The Standard Book of Spells Grade 4 (Miranda Goshawk)
The Standard Book of Spells Grade 5 (Miranda Goshawk)
The Standard Book of Spells Grade 6 (Miranda Goshawk)
The Standard Book of Spells Grade 7 (Miranda Goshawk)
Dark Charms and their Defense (Quentin Trimble)
Curses and Counter-Curses (Vindictus Viridian)
The Healer's Handbook (Hippocrates Smethwyck)
The Dragonkeeper's Guide (Harvey Ridgebit)
The Marauder's Manual of Mischief (Gellert Grindelwald)
The Jokester's Treasury of Jinxes (Zacharias Mumps)
A Beginner's Guide to Transfiguration (Emeric Switch)
Intermediate Transfiguration (Emeric Switch)
Advanced Transfiguration (Emeric Switch)
Practical Guide to Unbinding Viridian's Curses (Eldritch Diggory)

